models/order.js

```diff
@@ -40,6 +40,15 @@ orderSchema.statics.addEntries = function(orderId, newEntries) {
   })
 }
 
+orderSchema.statics.updateStatus = function(orderId, status) {
+  return this.findOne({ id: orderId }).then((order) => {
```

Comment:
Dude...didn't you run into an issue where findOneAndUpdate wouldn't work? I tried and wasn't able to, so had to do findOne...order.save() instead...